log tweak for failed pushes

tbg opened the pull request:
err.Error() is already pretty much the perfect
error message, especially since you usually see
it echoed from the DistSender, which prints the
same thing, so it helps if they look alike.

tamird commented on storage/range_command.go:
```diff
@@ -602,10 +602,11 @@ func (r *Range) InternalPushTxn(batch engine.Engine, ms *proto.MVCCStats, args *
+		err := proto.NewTransactionPushError(args.Txn, reply.PusheeTxn)
 		if log.V(1) {
-			log.Infof("failed to push intent %s vs %s using priority=%d", reply.PusheeTxn, args.Txn, priority)
+			log.Info(err.Error())
```
possibly stupid question: can you just `log.Info(err)`?

tschottdorf replied:
it's a valid point. It's going to be handled differently internally since the structured logging code does type assertions. But probably that's what we want more than losing structure by converting here already. I'll change it.

tbg committed:
log tweak for failed pushes

err.Error() is already pretty much the perfect
error message, especially since you usually see
it echoed from the DistSender, which prints the
same thing, so it helps if they look alike.

spencerkimball commented:
LGTM
